WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+import SwiftUI
+import WordPressKit
```

Comment:
[nit] unused imports
```suggestion
```

Reply:
I removed SwiftUI. I initially added because I was trying to add a SwiftUI preview, which didn't work.
WordPressKit is needed for `BlazeCampaignsSearchResponse`.